proxy/deploy_test.go

```diff
@@ -61,23 +60,31 @@ func Test_DeployFunction_Not2xx(t *testing.T) {
 			"function",
 			"image",
 			"language",
-			true,
+			deployTest.replace,
 			nil,
 			"network",
 			[]string{},
-			false,
+			deployTest.update,
 			[]string{},
 			map[string]string{},
 			FunctionResourceRequest{},
 		)
 	})
 
-	r := regexp.MustCompile(`(?m:Unexpected status: 404)`)
+	r := regexp.MustCompile(deployTest.expectedOutput)
 	if !r.MatchString(stdout) {
 		t.Fatalf("Output not matched: %s", stdout)
 	}
 }
 
+func Test_RunDeployProxyTests(t *testing.T) {
+	for _, tst := range DeployProxyTests {
+		t.Run(tst.title, func(t *testing.T) {
```

Comment:
Just wanted to ask why you did it this way instead of our usual coding style of test-cases and an in-line struct?

Reply:
I could change it so the test cases are defined in the `Test_RunDeployProxyTests()` method, but I like having `runDeployProxyTest()` as a separate function because it makes it easier to read